src/mount/mount.ceph.c

```diff
@@ -171,6 +171,7 @@ static char *parse_options(const char *data, int *filesys_flags)
 		} else if (strncmp(data, "secret", 6) == 0) {
 			if (!value || !*value) {
 				printf("mount option secret requires a value.\n");
+				free(saw_name);
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, in this case, it would be "mount.ceph: ".